src/input/source/hidraw/opineo.rs

```diff
@@ -162,18 +170,33 @@ fn translate_event(event: event::Event, touchpad_side: TouchpadSide) -> NativeEv
                 ),
             },
         },
+        event::Event::Trigger(trigg) => match trigg.clone() {
+            event::TriggerEvent::PadForce(_) => match touchpad_side {
+                TouchpadSide::Unknown => {
+                    NativeEvent::new(Capability::NotImplemented, InputValue::Bool(false))
+                }
+                TouchpadSide::Left => NativeEvent::new(
+                    Capability::Gamepad(Gamepad::Trigger(GamepadTrigger::LeftTouchpadForce)),
+                    normalize_trigger_value(trigg),
+                ),
+                TouchpadSide::Right => NativeEvent::new(
+                    Capability::Gamepad(Gamepad::Trigger(GamepadTrigger::RightTouchpadForce)),
+                    normalize_trigger_value(trigg),
+                ),
+            },
+        },
     }
 }
 
-/// List of all capabilities that the OrangePi NEO driver implements
+/// List of all input capabilities that the OrangePi NEO driver implements
 pub const CAPABILITIES: &[Capability] = &[
+    Capability::Gamepad(Gamepad::Trigger(GamepadTrigger::LeftTouchpadForce)),
+    Capability::Gamepad(Gamepad::Trigger(GamepadTrigger::RightTouchpadForce)),
     Capability::Touchpad(Touchpad::LeftPad(Touch::Button(TouchButton::Press))),
     Capability::Touchpad(Touchpad::LeftPad(Touch::Motion)),
     Capability::Touchpad(Touchpad::RightPad(Touch::Button(TouchButton::Press))),
     Capability::Touchpad(Touchpad::RightPad(Touch::Motion)),
 ];
 
-pub const OUTPUT_CAPABILITIES: &[OutputCapability] = &[
-    OutputCapability::ForceFeedback,
-    OutputCapability::LED(LED::Color),
-];
+/// List of all output capabilities that the OrangePi NEO supports
+pub const OUTPUT_CAPABILITIES: &[OutputCapability] = &[OutputCapability::ForceFeedback];
```

Comment:
Does the OrangePi Neo not support LEDs?

Reply:
It has RGB that is only controlled through button combo's on the device. OrangePi did not want to spend the money for an API for the MCU.